db/index.js

```diff
@@ -32,19 +32,19 @@ function sync(force=app.isTesting, retries=0, maxRetries=5) {
       // Don't do this auto-create nonsense in prod, or
       // if we've retried too many times.
       if (app.isProduction || retries > maxRetries) {
-        console.error(chalk.red(`********** database error ***********`))
-        console.error(chalk.red(`    Couldn't connect to ${url}`))
+        console.error(chalk.red(`********** database error ***********`));
```

Comment:
We actually don't need these semicolons- Emily mentioned that is Bones convention to not use semicolons.